Update hatecrime and privacy footer es i8n

Jkrzy opened the pull request:
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/542)

## What does this change?
Adding in latest spanish translations for
hatecrime question and privacy policy
footer.

## Screenshots (for front-end PR):
<img width="863" alt="Screen Shot 2020-05-26 at 9 52 15 PM" src="https://user-images.githubusercontent.com/3485564/82968571-9cf0de80-9f9b-11ea-814e-0a84050acaf0.png">

<img width="694" alt="Screen Shot 2020-05-26 at 9 50 49 PM" src="https://user-images.githubusercontent.com/3485564/82968581-a1b59280-9f9b-11ea-9477-51c673f475e4.png">

## Checklist:

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

Jkrzy commented:
🤔 Not sure why pa11y is failing on the link color contrast

LindsayYoung commented:
The error seems to be on the landing page, where I am seeing a contrast issue - but it isn't from this PR- I see it on dev too. 

![Screen Shot 2020-05-26 at 7 20 21 PM](https://user-images.githubusercontent.com/4406333/82970912-4a58f780-9f86-11ea-9a42-4f70c63c8de5.png)

Now that the pages are longer, I think we need to adjust the wait for commands for something at the end of the pages. I think that is why the a11y tests have been inconsistent lately.

Jkrzy commented:
Ahhhh thanks @LindsayYoung! A fix for that is included in #497

Jkrzy commented:
@LindsayYoung Pushed a small tweak to the `circleci` config and upload pa11y artifacts, results here are :interesting:.

Not sure why it appears to be only loading _some_ of the CSS
![pa11y](https://user-images.githubusercontent.com/3485564/83053034-19c49c80-a01e-11ea-9981-96177fbb0ab8.png)

LindsayYoung commented on .circleci/config.yml:
```diff
@@ -60,7 +60,7 @@ jobs:
             npm run test:a11y
       - store_artifacts:
-          path: /pa11y-screenshots
+          path: ./pa11y-screenshots
```
<3 Thank you I had been meeting to look in to this for sooo long!!!

Jkrzy commented:
Ooooo, it's because `runserver` doesn't serve static files unless `DEBUG = True`. So the pa11y tests in CI have maybe been running without styles

Jkrzy commented:
😌 

Rebased against `develop` to resolve conflicts and made updates necessary for pa11y suite to execute as expected and pass.

Includes: 
1. Adding of SR-only labels for accessible autocomplete
2. Modifying django settings/circleCI config so pa11y tests execute with css loaded
3. Removed a single pa11y test configuration in `crt-view` which felt a bit duplicative and was failing unexpectedly/inexplicable.

This can be replaced by #506 if desired

Jkrzy commented:
Replaced by #507
